metastore/iceberg-metastore/src/main/java/org/apache/drill/metastore/iceberg/components/tables/IcebergTables.java

```diff
@@ -42,16 +45,15 @@
  */
 public class IcebergTables implements Tables, MetastoreContext<TableMetadataUnit> {
 
-  public static final String STORAGE_PLUGIN = "storagePlugin";
-  public static final String WORKSPACE = "workspace";
-  public static final String TABLE_NAME = "tableName";
-  public static final String METADATA_KEY = "metadataKey";
-
   /**
    * Metastore Tables component partition keys, order of partitioning will be determined based
    * on order in {@link List} holder.
    */
-  private static final List<String> PARTITION_KEYS = Arrays.asList(STORAGE_PLUGIN, WORKSPACE, TABLE_NAME, METADATA_KEY);
+  private static final List<String> PARTITION_KEYS = Arrays.asList(
+    TableInfo.STORAGE_PLUGIN,
+    TableInfo.WORKSPACE,
+    TableInfo.TABLE_NAME,
+    MetadataInfo.METADATA_KEY);
```

Comment:
Thanks for cleaning up this code. If these are the four types, does it make sense to define them in an enum, with the string keys as a property of the enum? Maybe as part of `MetadataType`?

Reply:
Actually this a good idea to pull all Metastore column names into one place. I have created `MetastoreColumn` enum where all columns and their names are stored. Also made sure that in the code we refer only to `MetastoreColumn` and not to its string representation.